docs/utils/warnings.js

```diff
@@ -7,7 +7,7 @@ const warnMissingAccessToken = error => {
       "Don't forget to include the required scopes for the token: all repo scopes and read:org.",
       "After creating a token in GitHub, add it to a .env file in the root folder with GH_TOKEN=<YOUR TOKEN>",
     );
-  } else {
+  } else if (typeof error.missing === "undefined") {
```

Comment:
i think there is no need for that `else if` block, you can just add it as `if ((typeof error.missing === "undefined")` and that can be the first check before `GH_TOKEN` then it makes more sense I think  :)